crates/iota-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -131,12 +131,22 @@ pub(crate) type EncG = bls12381::G2Element;
 // retain a distinction anyway. If we need to support distributed object
 // storage, having this distinction will be useful, as we will most likely have
 // to re-implement a retry / write-ahead-log at that point.
-pub struct CertLockGuard(#[expect(unused)] MutexGuard);
-pub struct CertTxGuard(#[expect(unused)] CertLockGuard);
+pub struct CertLockGuard(#[allow(unused)] MutexGuard);
+pub struct CertTxGuard(#[allow(unused)] CertLockGuard);
```

Comment:
If this is no longer unused, then remove the tags altogether please

Reply:
Modified at [a1dbc6f](https://github.com/iotaledger/iota/pull/4757/commits/a1dbc6f2329904210cd6520dfc0478f3eb13fd7d). Note that for the `MutexGuard` the tag is still needed.

Reply:
Then it should be `expect`

Reply:
Enhanced the tag at [a8d3303](https://github.com/iotaledger/iota/pull/4757/commits/a8d33036b54d7a794c485d5feb9d9490490eb371).